### ISSUE TITLE
Better Handling of Adversary Proceedings in Bankruptcy Court

### ISSUE THREAD

**Issue by zvisrosen** [freelawproject/recap#278]
As of now, Recap and the Recap Archive don't handle adversary proceedings in bankruptcy court particularly well - very little metadata on the case is shown.

in bankruptcy you file under one of the common ones (7, 11, 13) in most cases.  There are also the exotic ones - 9, 12, and 15, which come up occasionally.  The chapters are literally chapters of the bankruptcy code (title 11, US code). That creates a main case which is often a giant ECF page - for GM and Chrysler they actually created a ECF-Mega server to handle the volume at SDNY.  Typically a bankruptcy case will lead towards confirmation of a plan of reorganization/liquidation, only exception being chapter 7, where the whole thing is a liquidation done by a court-appointed trustee.  Pretty much everything that's done is put on the docket.

In a few cases a case can be reopened years and even decades later, like the bankruptcy case mentioned as the oldest case on PACER, which was closed in 1931 and reopened in 2009.  Typically this is for failure to include assets in the schedules filed with the court.

Once bankruptcy is filed, the court has exclusive jurisdiction over all claims of the estate unless they lift the automatic stay and allow another court to proceed.  This means that typically actions to recover property of the estate - and actions against the estate - are moved/filed at the bankruptcy court.  So an adversary proceeding is actually a normal civil case litigated in the bankruptcy court and gets a double caption - the main case and the adversary case.  However, only the complaint from the AP is typically on the main docket - that's actually how you commence a AP within ECF (a motion to approve settlement of the AP also goes on the ECF page for the main bankruptcy case).

In a AP the claims are a mix of bankruptcy specific claims for turnover of assets under the bankruptcy code and more traditional civil claims by/against the estate.  That's essentially it.  The other main adversarial part of a bankruptcy (aside from the plan process, which can be quite contentious) is claims, which can be filed on ECF as well (in larger cases they're typically handled by a court-approved claims agent instead, which is why many bankruptcy cases have their own website and ECF copy.

In other words, an AP in bankruptcy court is very similar to a regular civil case in district court in terms of how it works, save for the relationship to the main bankruptcy case, with links between the cases on ECF appearing only really at the beginning/end of case.

In all bankruptcy ECF cases, APs can be found by hitting the "associated cases" button in ECF, perhaps a way to capture case relationships from here would be helpful as well.
